# `python:3.4.8-alpine3.4`

## Docker Metadata

- Image ID: `sha256:bb5317f1841c9181d715d5135605431548c84aaa61938fb2e44016433f792547`
- Created: `2018-04-20T03:30:27.139268641Z`
- Virtual Size: ~ 83.57 Mb  
  (total size of all layers on-disk)
- Arch: `linux`/`amd64`
- Command: `["python3"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `GPG_KEY=97FC712E4C024BBEA48A61ED3A5CA953F73C700D`
  - `PYTHON_VERSION=3.4.8`
  - `PYTHON_PIP_VERSION=10.0.0`
